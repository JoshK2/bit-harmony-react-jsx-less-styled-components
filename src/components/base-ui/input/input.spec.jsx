import React from 'react';
import testRenderer from 'react-test-renderer';
import { BasicInput } from './input.composition';

describe('Input component', () => {
  it('renders correctly', () => {
    testRenderer.create(<BasicInput />);
  });
  it('snapshot test', () => {
    const component = testRenderer.create(<BasicInput />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
