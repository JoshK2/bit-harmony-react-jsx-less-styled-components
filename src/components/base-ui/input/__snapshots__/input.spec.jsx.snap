// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component snapshot test 1`] = `
<input
  className="sc-bdfBwQ jyVkcv"
/>
`;
